src/dev/register_git_hook/register_git_hook.js

```diff
@@ -51,8 +52,48 @@ function getKbnPrecommitGitHookScript(rootPath) {
     command -v node >/dev/null 2>&1
   }
   
+  has_nvm() {
+    command -v nvm >/dev/null 2>&1
+  }
+  
+  try_load_node_from_nvm_paths () {
```

Comment:
Is this necessary? Doesn't nvm create a link within the paths you have defined in `extend_user_path`?

Reply:
@tylersmalley at least on my local setup it didn't. Try to install Git-Tower for example, run this register_git_hook to install the pre-commit hook without this nvm paths setup and try to commit, u'll not be able to commit.